### PR TITLE
feat: add @comet/eslint-plugin-react package

### PR DESCRIPTION
**Pull request by manuelblum**
- add @comet/eslint-plugin-react package
- add custom rule `id-allow-list-react-intl`

### CODE REVIEW

File: packages/eslint-plugin-react/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "name": "@comet/eslint-plugin-react",
+    "version": "0.0.0",
```
[the remark this answers is not in the document]
Comment (johnnyomair) [reply packages/eslint-plugin-react/package.json:3]: And it needs to be added to lerna.json

**Review by johnnyomair — COMMENTED** [review]
Thanks!

For our projects we'd like to have a rule that prevents developers from using `comet.*` messages and recommends to use `messages` from `@comet/admin` instead. How would you achieve this?

**Comment by kaufmo** [comment]
one generic question, why is this package/plugin named with react, what it does is to force to remove the intl prefix that has nothing todo with react or am I wrong? Should we not intend on working on a @comet eslint plugin only?

**Comment by johnnyomair** [comment]
@thomasdax98 should we add an ESLint rule to validated message IDs?

**Comment by johnnyomair** [comment]
Closing this for now. If we decide to add the rule, we'll start fresh from `main`.

**Pull request closed without merging**